### DISCUSSION TITLE
reordable not working in table.

### DISCUSSION THREAD

**Discussion by yuvrajtimalsina** [filamentphp/filament#12910] [Help]
### Package

Table builder

### Package Version

v3.2.81

### How can we help you?

I have set  reordable in table as below :

```
->reorderable('sort')
->defaultSort('sort')
```

I have `Model::unguard();` in AppServiceProvider and `$table->unsignedInteger('sort')->default(0);` column in table.

**Comment by leandrocfe** [comment]
Could you show the issue?

**Reply by yuvrajtimalsina** [reply]
reordable icon is not displayed in table, thats the issue for now. I will try to replicate the issue in new repo and share.

**Reply by yuvrajtimalsina** [reply]
@leandrocfe seems the issue was related to reorder generation in policy by filament shield although not defined in config. Thank you
